lib/capistrano/tasks/assets.rake

```diff
@@ -62,15 +62,29 @@ namespace :deploy do
       end
     end
 
+    def manifest_path(filename)
+      release_path.join('public', fetch(:assets_prefix), filename)
+    end
+
+    def with_backup_and_manifest_paths &block
```

Comment:
As @kirs mentioned [below](https://github.com/capistrano/rails/pull/112/files#r28380633), why does it accept block instead of returning them?

Also missing parentheses around arguments.